README.md

```diff
@@ -97,6 +97,35 @@ class MyApp < Sinatra::Base
 end
 ```
 
+### If using Plain Ruby
+
+Rollbar isn't dependent on Rack or Rails for most of its functionality. In a regular script, assuming you've
+installed the rollbar gem:
+
+ 1. Require rollbar
+ 2. Configure rollbar
+ 3. Send Rollbar data
+
+```ruby
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = "SERVER_POST_ACCESS_TOKEN"
+  # Other Configuration Settings
+end
+
+begin
+  Rollbar.debug "Running Script"
+
+  run_script ARGV
+
+  Rollbar.info "Script ran successfully"
+rescue Exception => e # Never rescue Exception *unless* you re-raise in rescue body
+  Rollbar.log e
```

Comment:
I think it'd be desirable having here `Rollbar.error()` insead of `Rollbar.log`.